web/src/installers/versions.js

```diff
@@ -78,7 +78,7 @@ module.exports.InstallerVersions = {
     "19.03.4",
     "18.09.8",
   ],
-  containerd: ["1.5.11", "1.5.10", "1.4.13", "1.4.12", "1.4.11", "1.4.10", "1.4.9", "1.4.8", "1.4.6", "1.4.4", "1.4.3", "1.3.9", "1.3.7", "1.2.13"], // cron-containerd-update
+  containerd: ["1.6.4", "1.5.11", "1.5.10", "1.4.13", "1.4.12", "1.4.11", "1.4.10", "1.4.9", "1.4.8", "1.4.6", "1.4.4", "1.4.3", "1.3.9", "1.3.7", "1.2.13"], // cron-containerd-update
```

Comment:
I know we used to have code to make 1.4.6 the first entry here despite not being the most recent - should we do the same with 1.6.x to put it at the end?

Or a fix to weave, one of the two